.github/workflows/documentation.yml

```diff
@@ -0,0 +1,38 @@
+name: "Documentation"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+    paths:
+      - .github/workflows/documentation.yml
+      - docs/**
```

Comment:
Yamllint says you should quote every string with double quotes. that includes these 2 and a few others below as well.

Reply:
You need to address this comment in order to fix the build.